### ISSUE TITLE
fadvise/drop test fail under aarch64

### ISSUE THREAD

**Issue by fundawang** [util-linux/util-linux#2398]
I'm building util-linux 2.39/2.39.1 under openanolis (fedora rawhide based distribution), it failed when testing fadvise/drop:
https://[build.openanolis.cn/kojifiles/work/tasks/8775/678775/build.log

The drop test was passed under x64 environment.

**Comment by t-8ch** [comment]
Yes this should be the same. Maybe we should backport #2251 to the stable branch.

**Comment by xry111** [comment]
> Yes this should be the same. Maybe we should backport #2251 to the stable branch.

IMO it would be better to backport it if 2.40 won't be released soon.
